cdk/lib/frontend.ts

```diff
@@ -136,7 +142,17 @@ export class Frontend extends GuStack {
         },
       },
       monitoringConfiguration: {
-        noMonitoring: true,
+        snsTopicName: "reader-revenue-dev",
+        http5xxAlarm: {
+          alarmName: alarmName(
+            "support-frontend instances are returning 5XX errors"
```

Comment:
Minor: I would suggest `support-frontend is returning 5XX errors` (because technically it might be the load balancer rather than the instances!)

Reply:
Conveniently the suggested name is also unique, which means AWS will allow you to create it 😀

Reply:
Yeah great suggestion - I've pushed that up now!